crates/nu-cmd-extra/src/extra/mod.rs

```diff
@@ -1,9 +1,26 @@
 mod bits;
 
-pub use bits::add_bits_decls;
-
 use nu_protocol::engine::StateWorkingSet;
 
 pub fn add_extra_decls(working_set: &mut StateWorkingSet) {
-    add_bits_decls(working_set);
```

Comment:
so this will now enable us to pull in individual bits commands instead of the all or none approach we had before...

In other words before you got *all of them* or *none of them* ?

Reply:
i've made it such that it's exactly the same as in the `default_context.rs` of `nu-command`, so yes you can now pull in individual commands from `mod.rs`!

but in the end, you already could by modifying the `mod.rs` of `nu-cmd-extra/src/extra/` :yum: 

i just thought it might be easier to have them all in the same place :thinking: